docs/reference/functions/arithmetic/manhattan.html

```diff
@@ -63,6 +63,14 @@
 :   The call `pedantic(manhattan)(x,y)`  or `pedantic(manhattan)(x,y,z)` computes the sum of the absolute values of its parameters
     returning $\infty$ as soon as one of its parameter is infinite, regardless of possible `Nan` values.
 
+[`derivative`](../decorators.html#derivative)
+:   **Required header: **<script type="preformatted">` #include <eve/function/derivative/average.hpp>`</script>
+
+:   The expression `derivative_1st(manhattan)(x,y)` and `derivative_2nd(manhattan)(x,y)` computes the partial
+    derivatives of $f$, where $f$ is the function $(x,y) \rightarrow \ |x|+|y|$. Of course 3 analogous calls
```

Comment:
For FMA, you specify it clearly, so why not here instead of this "Of course..." ?

Reply:
idem

Reply:
idem